src/lib/merkle_ledger_tests/test_mask.ml

```diff
@@ -82,7 +82,9 @@ module Make (Test : Test_intf) = struct
 
   let dummy_location = Test.Location.Account dummy_address
 
-  let dummy_account = Quickcheck.random_value Account.gen
+  let dummy_account =
+    let account = Quickcheck.random_value Account.gen in
+    { account with token_id = Token_id.default }
```

Comment:
Do you mean, use a non-default token here? The ledger should never have that in mainnet.

The original code generated the same result with the `dev` profile.

Reply:
But this is for berkeley, right?

Reply:
Yes, but this code is to test masks and their parents, whether it's a token account isn't so important.

The code comes from the same commits against `compatible`, so we won't get a conflict when it's merged to `develop`.

Reply:
>whether it's a token account isn't so important.

What was the reason to set it to the default token then?

If the commits are added as is from compatible and conflict resolutions/changes are made in newer commits (or merge commit) then there won't be any conflicts, right? Also, `develop` should have same berkeley changes unless for some reason we want it differently in `develop`

Reply:
Setting to the default token was because in non-`dev` profiles, the generated token could be other than the default.  That meant that these tests failed using other profiles, because `Account.check` rejected those tokens.

That's fixed in #13347; there, `Token_id.gen` always generates the default token.

Since this is merged, I can do some cleanup after #13347 is merged.